load_blazegraph_test.sh

```diff
@@ -10,16 +10,85 @@ BASEURI="http://ld4p-test.stanford.edu/"
 # vars above this line may need to change to process other data
 #------------------------------------------------
 
-BG_URL="http://localhost/blazegraph/"
-TEST_NAMESPACE=test
```

Comment:
I was thinking the test namespace would be cleaner to manage objects merely for this test.